composer.json

```diff
@@ -39,7 +39,7 @@
         "phpstan/extension-installer": "^1.3",
         "phpstan/phpstan-strict-rules": "^1.5",
         "phpunit/phpunit": "^9.6",
-        "rector/rector": "1.2.0"
+        "rector/rector": "1.2.1"
```

Comment:
Remove this unrelated change.

Reply:
Rector is already v1.2.1 on `develop`.
https://github.com/codeigniter4/shield/blob/develop/composer.json#L42

```
git fetch upstream
git rebase upstream/develop
```

Reply:
done